### ISSUE TITLE
Most of the (first) stairs or theshholds block ur way

### ISSUE THREAD

**Issue by Zerroc1** [SunwellWoW/Sunwell-TBC-Bugtracker#1398]
Decription: 
Espacially on the first stair or on a threshhold the character get stuck. Realised it many times in Elwynn/Stormwind/Westfall

How it works: 
U stuck when u want to cross the threshhold and most of the time you have to jump to pass it

How it should work: 
just walk over the stairs, no block

Source (you should point out proofs of your report, please give us some source): 
Entrance to Northshire abbey (first stair)

**Comment by akrom23** [comment]
I believe it's clientside.

**Comment by Roarek** [comment]
I don't believe it's client side, several friends are having these issues in several different locations with steps/ledges but no hitches on other TBC servers we've played on using same client. Occurs in many places in Stormwind too (the entry gates where Ony's head goes on the left side, some steps up to various shops). This is happening with human female characters, I haven't tested with other models to see if that's somehow related.

Some examples of surfaces that should be smoothly stepped over but will hitch (there are many many more):

https://puu.sh/CbCdv/9ebaef61ee.mp4

https://streamable.com/basz0

https://streamable.com/41vdh

Slaughtered Lamb crypts: https://streamable.com/jlmsu

**Comment by foxxhound** [comment]
Bumping this. Also agreeing that this is indeed non-client side. There's multiple places where stairs are not allowing you to smoothly go up them. Even leaving the War Room in Stormwind will cause this. Any small sharp rise in terrain that causes the character to rise is having issues.
